src/clients/python/client_build.py

```diff
@@ -0,0 +1,17 @@
+from cffi import FFI
+ffibuilder = FFI()
+
+# For the cdef, we cheat by just pulling in our C header and removing the ifdefs and includes :)
+c_header = [line for line in open("../c/tb_client.h").readlines() if not line.startswith("#")]
+
+# Also, replace the definition for __uint128_t because cffi doesn't support it yet.
+t_index = c_header.index("typedef __uint128_t tb_uint128_t;\n")
+c_header[t_index] = "typedef struct tb_uint128_t {uint64_t high; uint64_t low;} tb_uint128_t;\n"
```

Comment:
That `{uint64_t high; uint64_t low;}` looks great!
That's the same strategy the Java client uses, with an external helper converting from array, guid, BigInteger, etc.